start.py

```diff
@@ -44,5 +44,11 @@ def start_jenkins():
 
 
 if __name__ == "__main__":
-    proc = start_jenkins()
-    sys.exit(proc.wait())
+    proc = None
+
+    try:
+        proc = start_jenkins()
```

Comment:
but then, the proc won't be returned, so it will be None. So you won't kill anything, no ?